pkg/reporting/cookbooks.go

```diff
@@ -378,7 +400,14 @@ func (cbr *CookbooksReport) downloadCookbookArtifact(item cookbookItem) *Cookboo
 	if cbr.RunCookstyle {
 		err = cbr.cookbookArtifacts.DownloadTo(item.Name, item.CBAIdentifier, cbr.cookbooksDir)
 		if err != nil {
-			cbState.DownloadError = errors.Wrapf(err, "unable to download cookbook %s", item.Name)
+			name := ""
+			if cbr.Anonymize {
+				name = hashString(item.Name)
+			} else {
+				name = item.Name
+			}
+
+			cbState.DownloadError = errors.Wrapf(err, "unable to download cookbook %s", name)
```

Comment:
`err` itself - as a string - may contain reference to one or more specific cookbook file paths.

Reply:
It is following the existing pattern as in downloadCookbook(), not updated with this PR. Also, I think wrapped error includes the original `err` details.
On another note I did improve the if / else above it with latest commit.